src/openms/include/OpenMS/FILTERING/ID/IDFilter.h

```diff
@@ -690,8 +690,8 @@ namespace OpenMS
        @brief Extracts all unique peptide sequences from a list of peptide IDs
 
        @param peptides Input
-       @param sequences Output
        @param ignore_mods Extract sequences without modifications?
+       @return sequences 
```

Comment:
I am not sure about this one. In general, a void function does not return anything. Here the function is used to manipulate an Object given to the function as a parameter, so I guess the @param is correct.

Reply:
Fixed.